packages/sitecore-pipelines/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2020",
```

Comment:
Why the jump to es2020 here? Looks like the rest stuck with es6.

Reply:
This was to handle the dynamic imports, as they are not supported in es6.

Reply:
Ah, that's right